### PR TITLE
Active Admin

### PR DESCRIPTION
**Pull request by FVolonnino**
https://trello.com/c/E3njpzaE/10-instalar-y-configurar-active-admin

### CODE REVIEW

File: app/models/rent.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Rent < ApplicationRecord
-  belongs_to :user
-  belongs_to :book
+  belongs_to :user, optional: true
```
Comment (Gmarn) [inline app/models/rent.rb]: Humm, why this change?

File: app/models/rent.rb [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+  belongs_to :book, optional: true
```
Comment (Gmarn) [inline app/models/rent.rb]: same

File: app/models/rent.rb (hunk 1 above)
Comment (FVolonnino) [reply app/models/rent.rb]: Active admin didn't let me create Rents without that change. It kept giving me an error, I think it was "must exist" or something like that.

File: Gemfile [hunk 3]
```diff
@@ -59,6 +59,10 @@ gem 'sidekiq', '~> 5.2.2'
 
 gem 'redis-rails', '~> 5.0.2'
 
+gem 'dotenv-rails', '~> 2.5.0', groups: [:development, :test]
```
[the remark this answers is not in the document]
Comment (FVolonnino) [reply Gemfile]: I think I didn't do the rebase on this branch, I'll do it now and it should be fixed.

File: db/seeds.rb [hunk 4]
```diff
@@ -5,3 +5,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+AdminUser.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```
Comment (Gmarn) [inline db/seeds.rb]: You forgot the EOF (End Of File), here
